index.html

```diff
@@ -2,13 +2,238 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="icon" href="public/icon-list.png" alt="list-icon" />
```

Comment:
Is there a reason for this ???,if not lets remove it

Reply:
@Yinkusadel Thank for the Review, I am making use of this if you open the Todo List App on Netlify the The Front-End Mentor Logo has been replaced by that